src/sentry/tagstore/snuba/backend.py

```diff
@@ -198,9 +202,8 @@ def __get_tag_keys_for_projects(
         return set([
             ctor(
                 key=key,
-                values_seen=data['values_seen'],
-                count=data['count'],
-            ) for key, data in six.iteritems(result) if data['values_seen']
+                count=count,
+            ) for key, count in six.iteritems(result)
```

Comment:
I think this is going to potentially break because of some (probably bad in retrospect) cleverness from the `snuba.query` method.

If you ask for only one aggregate column, it just uses the value of the aggregate as the leaf node. eg
```
{"key1": 1, "key2": 100}
```
But if you have multiple aggregates, it puts them in there by name. eg.
```
{"key1": {"count": 1, "values_seen": 1}, "key2":  {"count": 100, "values_seen": 25}}
```
so the structure of the response object actually changes here based on whether there are 1 or 2 aggregate columns. Sorry I know this is bad, I added it once to try and simplify TSDB, but it has probably caused many more headaches than it has solved.

Reply:
I guess we should at least test for both cases.

Reply:
Right, handled that case now. I wonder if we should look to refactor and remove this behaviour?